### PR TITLE
Support for wildcard on localization info

### PR DESCRIPTION
**Pull request by dsaradini**
Added tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+coverage:
+	coverage run --source=crowdin test
+	coverage html
+	open htmlcov/index.html
```
Comment (brutasse) [inline Makefile:6]: Maybe remove the open line?

File: test [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env python
+
+from tests.main import main
+
+if __name__ == '__main__':
+    main()
```
Comment (brutasse) [inline test]: You can add `test_suite='tests'` in `setup.py` and avoid that boilerplate.

See https://github.com/brutasse/rache/blob/master/setup.py#L56 for an example.

**Comment by brutasse** [comment]
You need to add `recursive-exclude tests *` to `MANIFEST.in`, otherwise you'll get the tests installed with the package under the `tests` namespace. To check that the manifest is correct, run `python setup.py sdist` and look at the generated tarball. The test files shouldn't appear here.

**Comment by dsaradini** [comment]
Ok removed the "test", I'll use "pyhton setup.py test".

File: README.rst [hunk 3]
```diff
@@ -40,10 +44,23 @@ following structure::
...
+                    "fr": "locale/en/LC_MESSAGES/",
+                    "de": "locale/de/LC_MESSAGES/",
+                    "it": "locale/it/LC_MESSAGES/"
+                }
             }
         ]
     }
 
+The second entry in the json is to use a complete directory as translation source / destination.
+Usefull for documentation. DO NOT FORGET the trailing slash for directories.
```
Comment (brutasse) [inline README.rst]: Useful

File: requirements.txt [hunk 4]
```diff
@@ -1 +1 @@
-requests
+requests==1.2.3
```
Comment (brutasse) [inline requirements.txt]: This is a bit restrictive. The client should work fine with other versions. (requirements.txt go to install_requires)

Comment (dsaradini) [inline requirements.txt]: mmm... I changed that because the moch system was broken in 1.1.X.

File: tests/__init__.py [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                json.dumps(self.kwargs.get('info', "")).encode('utf-8')
+            )
+            response.status_code = 200
+            return response
+        if url.endswith('/test-project/download/all.zip'):
+            response = create_zip_response(self.kwargs.get('zip', {}))
+            response.status_code = 200
+            return response
+        else:
+            assert NotImplemented
```
Comment (brutasse) [inline tests/__init__.py]: `raise NotImplementedError`

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,17 @@
+language: python
+
+python:
+  - "2.7"
+
+install:
+  - pip install -r requirements-dev.txt
+
+before_script:
+  - pep8 tests
```
Comment (brutasse) [inline .travis.yml]: Why not flake8? Also maybe lint the crowdin namespace?

File: tests/main.py [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import unittest
+
+
+def main():
+    print "Running tests"
+    suite = unittest.TestLoader().discover("tests")
+    unittest.TextTestRunner(verbosity=2).run(suite)
+
+if __name__ == "__main__":
+    main()
```
Comment (brutasse) [inline tests/main.py]: This file shouldn't be needed.

Comment (dsaradini) [inline tests/main.py]: removed.

File: README.rst (hunk 3 above)
Comment (dsaradini) [inline README.rst]: corrected

**Pull request merged**